source/index.html.md

```diff
@@ -966,21 +966,20 @@ Examples:
 
 ### Stop recurring status
 
-On any recurring transaction issures have the ability to issue a <i>stop
-recurring</i>, canceling the concent of the card holder, towards a merchant, to
-initiate transactions on their behalf.
+On any recurring transaction, issuers can respond with <i>stop recurring</i>,
```

Comment:
Incorrect. Stop is something scheme does, not the issuer. That being said, I acknowledge my very long and complicated sentence.